src/styles/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
```

Comment:
please provide card animation on hover: readme `❗️❗️❗️ MANDATORY if you're familiar with the transform and transition`